src/biogeochem/CNNStateUpdate1Mod.F90

```diff
@@ -175,6 +175,7 @@ subroutine NStateUpdate1(num_soilc, filter_soilc, num_soilp, filter_soilp, &
 
          ! phenology: litterfall and retranslocation fluxes
          ns_veg%leafn_patch(p)    = ns_veg%leafn_patch(p)    - nf_veg%leafn_to_litter_patch(p)*dt
+         ns_veg%leafn_patch(p)    = ns_veg%leafn_patch(p)    - nf_veg%leafn_to_biofueln_patch(p)*dt
```

Comment:
This flux is not set to 0 if running with use_crop false, so it isn't safe to reference it here. I feel the best thing to do is to move this line down into the `if (ivt(p) >= npcropmin)` conditional block below (near the reference to `livestemn_to_biofueln`).